src/react-app-env.d.ts

```diff
@@ -19,6 +19,7 @@ interface Window {
       isCoinbaseBrowser: boolean
       isCoinbaseWallet: boolean
       isMetaMask: boolean
+      close?: () => void
```

Comment:
revert this

Reply:
https://github.com/KyberNetwork/kyberswap-interface/pull/1577#discussion_r1214423852